subsys/nrf_security/src/ssf_secdom/bounce_buffers.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <nrf.h>
+#include <nrfx.h>
+
+#define CACHE_DATA_UNIT_SIZE (DCACHEDATA_DATAWIDTH * 4)
+
+#ifdef CONFIG_PSA_SSF_CRYPTO_CLIENT_OUT_BOUNCE_BUFFERS
```

Comment:
Do we need this? Wouldn't it suffice to use wrap the invocation of these functions with `IS_ENABLED` on the caller side?

Reply:
bounce_buffers is very simple.

I'd rather make bounce_buffers a little bit more complicated than to make the users code a little more complicated.

This is also consistent with Karsten's implemenation.

Reply:
It's also consistent with the Zephyr cache API, which is the "other" API that the caller uses.

So the caller will be more consistent if we deal with this in bounce buffer.

We won't have IS_ENABLED for the bounce_buffers, and nothing for the cache flushing.